### ISSUE TITLE
[Tier 3] Disability Claim Application Issues - SF 06194274

### ISSUE THREAD

**Issue by Kimberley2019** [department-of-veterans-affairs/va.gov-team#34289]
## Issue Description
_Caller is having issue completing and submitting claim application.  Review encrypted email and details below._

---
## Tasks
- [ ] _Review provided details in ticket_
- [ ] _Review additional information from encrypted email_
- [ ] _Advise if additional information is required for review and resolution_
- [ ] _Contact veteran and set up shadowing call if needed_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by Kimberley2019** [comment]
Case Notes
Veteran is trying to submit his disability increase but when he tries to submit it tells him some of his information is "missing or invalid" in his supporting evidence. He's reviewed the information and couldn't find anything wrong with it. His application ID is #37506933. It's been going on for three days.

**Comment by Kimberley2019** [comment]
@alsleaders encrypted email was sent

From: Daniels, Kimberley N. (Oddball, Inc.) 
Sent: Friday, December 17, 2021 11:06 AM
To: Leaders, Amanda L. (Government CIO) <Amanda.Leaders@va.gov>
Cc: Henry, Veronica (GovernmentCIO) <Veronica.Henry@va.gov>
Subject: FW: [EXTERNAL] Re: Application submission issue follow up

Good Morning Amanda,

I wanted to pass this along to you since it contains screenshots that contain PII.  The ZH ticket number is below.

[Tier 3] Disability Claim Application Issues - SF 06194274 · department-of-veterans-affairs/va.gov-team (zenhub.com)



Kind Regards,

Kimberley

**Comment by alsleaders** [comment]
I'm blocked on this until I can research Application Id 3750693.

**Comment by Mottie** [comment]
I've merged in a possible fix for this issue. It will be available in production tomorrow after 3pm ET

**Comment by jason-gcio** [comment]
Thanks @Mottie - lets circle back on this and we can write a new ticket if needed
cc: @alsleaders, @vhenry07

**Comment by michelpmcdonald** [comment]
@Kimberley2019 - please hold off on notifying the vet, we are currently testing the fix Robin did yesterday.

**Comment by Mottie** [comment]
So it appears that the code change will not fix this issue. @michelpmcdonald shared the data with me and on the `/disability/file-disability-claim-form-21-526ez/supporting-evidence/evidence-types` page the "VA medical records" checkbox was checked

<details><summary>Supporting evidence page</summary>


![Screen Shot 2022-01-13 at 11 14 01 AM](https://user-images.githubusercontent.com/136959/149377234-413f3810-18e7-45d9-afee-7bdbd3e75e2b.png)</details>

But somehow the next page `/disability/file-disability-claim-form-21-526ez/supporting-evidence/va-medical-records` is completely blank.

<details><summary>VA medical records</summary>


![Screen Shot 2022-01-13 at 11 14 26 AM](https://user-images.githubusercontent.com/136959/149377230-8f108abe-8574-4baf-a078-3ec86fb4695d.png)</details>

The VA medical facilities page still needs to be filled it. I don't know how the form let the Veteran get past this page.

Please let the Veteran know that this page needs to be filled in, or they can un-check the "VA medical records" checkbox on the previous page.

**Comment by michelpmcdonald** [comment]
@Kimberley2019 - please review Robin's notes screenshots and notes from above.  "The VA medical facilities page still needs to be filled it" ^.

**Comment by Kimberley2019** [comment]
Reached out to caller to see if they can try task again.

**Comment by jason-gcio** [comment]
We can reference this ticket and draft a new ticket if there is new work to be done here.
